components/builder-api-client/src/lib.rs

```diff
@@ -35,8 +36,10 @@ pub use error::{Error, Result};
 use std::io::Read;
 use std::path::Path;
 
+use builder_core::data_structures::UnsuccessfulJobGroupPromote;
```

Comment:
My intent, as I laid out in a comment in that file, was to have a place to put common data structures that are required by both the client and the server, e.g. the server generates a JSON response that represents some data structure and the client needs to parse that and re-construct it.  `data_structures` felt like it accurately represented the intent to me, but I'm certainly open to suggestions.